docs/reference/settings/security-settings.asciidoc

```diff
@@ -1380,23 +1276,21 @@ a PKCS#12 container includes trusted certificate ("anchor") entries look for
 
```

Comment:
If I'm understanding correctly, that PKCS#11 already appears under the appropriate HTTP and Transport sections. So unless there are other contexts where there would be PKCS#11 settings, I think it can be removed from the "default values..." section.

Reply:
I think we reference PKCS#12 in those sections but not PKCS#11

Reply:
Scratch that. I am wrong, we already have it there. So it can be removed from the defaults section